build.rs

```diff
@@ -1,9 +1,28 @@
-#[cfg(not(windows))]
-fn main() {
-    pkg_config::find_library("libftdi1").unwrap();
-}
+use bindgen;
+use std::env;
+use std::path::PathBuf;
 
-#[cfg(windows)]
 fn main() {
+    #[cfg(not(all(windows, target_env="msvc")))]
+    pkg_config::Config::new().atleast_version("1.4").probe("libftdi1").unwrap();
+    #[cfg(all(windows, target_env="msvc"))]
     vcpkg::find_package("libftdi1").unwrap();
+
+    let bindings = bindgen::Builder::default()
+        .header("wrapper.h")
+        .default_enum_style(bindgen::EnumVariation::Rust{ non_exhaustive : true })
```

Comment:
There is an interesting problem here: according to the recent discussion in rust-lang/rust-bindgen#1554, this `#[non_exhaustive]` would not help with unknown variants if a newer libftdi1 version returns them (it's still UB to have an enum with a value `rustc` does not know about). We may want to look into the `newtype` style of enums, it looks interesting.